tests/t/lib/ProFTPD/Tests/Modules/mod_ifsession.pm

```diff
@@ -1379,7 +1379,7 @@ sub ifgroup_displaylogin_bug3882 {
     AuthGroupFile => $setup->{auth_group_file},
     AuthOrder => 'mod_auth_file.c',
 
-#    DefaultRoot => '~',
+    DefaultRoot => '~',
```

Comment:
Reverts some additional changes for https://github.com/proftpd/proftpd/pull/1641 which should not have been committed.